serveradmin/settings.py

```diff
@@ -17,6 +17,8 @@
 
 DEBUG = env('DEBUG', default=False)
 
+ALLOWED_HOSTS = env('ALLOWED_HOSTS', default=[])
```

Comment:
`ALLOWED_HOSTS` in env is a string, but in python it needs to be a list. How is that working?

Reply:
I don't know you would have to check the implemention of the library or Django. 

I tested it with `ALLOWED_HOSTS="0.0.0.0"` (see .env.dists) and it seems to properly allow the correct hosts.

Reply:
Most probably that's then some magic of the `django-environ` library.